projects/pgai/pgai/vectorizer/features/features.py

```diff
@@ -22,3 +22,12 @@ def disable_vectorizers(self) -> bool:
     def worker_tracking(self) -> bool:
         """If the worker tracking feature is supported by the extension."""
         return self.ext_version > version.parse("0.8.0")
+
+    @cached_property
+    def loading_retries(self) -> bool:
+        """If the loading retries feature is supported by the extension.
+
+        The feature includes changes in the way we fetch_work from the
+        queueing tables, and also how we handle the retries.
+        """
+        return self.ext_version > version.parse("0.9.0")
```

Comment:
@cevian this will conflict with your in-flight change: https://github.com/timescale/pgai/pull/573

Reply:
good catch, I guess we could test if the function to create the _vectorizer_q_failed tables is available.